linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/CoreWasmLib.scala

```diff
@@ -1028,6 +969,31 @@ final class CoreWasmLib(coreSpec: CoreSpec) {
     fb.buildAndAddToModule()
   }
 
+  /** `getClassOfNullable: (ref null typeData) -> (ref null jlClass)`.
+   *
+   *  Like `getClassOf`, but returns `null` if the argument is `null`.
+   */
+  private def genGetClassOfNullable()(implicit ctx: WasmContext): Unit = {
```

Comment:
This seems to be only used by `Class_componentType`. Wouldn't it be better (code-size wise) to move the get's of the class data also into this helper?

IMO this would also make it clearer what this is used for / why this is here.